.travis.yml

```diff
@@ -71,6 +71,6 @@ jobs:
         - source ./apps/ci/ci-before_install.sh
       install:
         - source ./apps/ci/ci-install.sh ON
-        - source ./apps/ci/ci-install-modules.sh
+        # - source ./apps/ci/ci-install-modules.sh For test no need module
```

Comment:
ok but don't forget to put them back o_0

Reply:
Yes yes, hah